pvl_hsu_soiling.m

```diff
@@ -0,0 +1,351 @@
+function SR = pvl_hsu_soiling(Time, Rain, RainThresh, Tilt, PM2_5, PM10, varargin)
```

Comment:
Reorder name to `pvl_soiling_hsu`  what does "hsu" stand for?

Reply:
hsu = Humboldt State University. I asked Liza if she had a preferred name for the model (e.g. Boyle-Coello), and she said that the given name "pvl_hsu_soiling" was acceptable. 

I am open to changes, though, this was really just my first go at naming the model.

Reply:
"hsu" it is, would be nice to find a way to work Humboldt state University into the notes.

Reply:
Really great point. I added a whole section between the Output Parameters and the References:
% Background and recognition:
%   This soiling model was developed by Liza Boyle and Merissa Coello at
%   Humboldt State University (HSU). Many thanks to Liza and Merissa for
%   model development, initial code, and subsequent code and performance
%   reviews.